README.md

```diff
@@ -17,138 +19,143 @@ Before getting started with this project, please get yourself familiar with the
+
+    1. From within the `yocto` directory checkout `meta-azure-device-update` at the version of Device Update you plan to use in your test. 
 
     ```shell
     git clone --branch $adu_release http://github.com/azure/meta-azure-device-update
     ```
-
-- Clone the IoT Hub Device Update Delta meta layer.
-
+    2. From within the `yocto` directory checkout `meta-raspberrypi-adu` at the version of Device Update you plan to use in your test. 
     ```shell
-    git clone --branch $adu_release http://github.com/azure/meta-iot-hub-device-update-delta
+    git clone --branch $adu_release http://github.com/azure/meta-raspberrypi-adu
     ```
-
-- Clone the Raspberry Pi with ADU meta layer.
-  
+    3. (optional) If you're planning on using delta updates you can checkout the `meta-iot-hub-device-update-delta` layer that integrates that functionality into that agent. Please read the `README.md` if you want to know more. 
     ```shell
-    git clone --branch $adu_release http://github.com/azure/meta-raspberrypi-adu
+    git clone --branch $adu_release http://github.com/azure/meta-iot-hub-device-update-delta
     ```
 
-### How To Build The Project Locally
+Next we move on to how to build the project assuming you've setup the project like above instructions. If you don't follow the setup instructions you will have to make modifications to `scripts/build.sh` to make sure the build works. 
+
+### Building The Project Locally
 
 #### Install Build Dependencies and Tools
 
-For more information, see [Yocto Project Quick Build](https://docs.yoctoproject.org/brief-yoctoprojectqs/index.html#yocto-project-quick-build)
+For more information on the Yocto build system, the open embedded base image, and example builds please see [Yocto Project Quick Build](https://docs.yoctoproject.org/brief-yoctoprojectqs/index.html#yocto-project-quick-build). 
 
-```sh
-# Install build dependencies
-sudo ./scripts/install-deps.h
+Please look into `scripts/install-deps.sh` to determine what you may need to integrate into your build as you move forward with the project. 
 
-# Checkout a desired 'poky' branch
-cd yocto/poky
-git fetch
-git checkout -t origin/honister -b my-honister
-git pull
 
-# Initialize build environment
-source oe-init-build-env
+1. From the `project_root` please execute the following command in your terminal
+```shell
+sudo ./scripts/install-deps.h
 ```
 
+### Creating the Private Key for Sw Update Signing
+
+To create the `*.swu` file you will need to provide the build system with a private key and password file so that it can sign the generated image and then create the Sw Update file. This is REQUIRED for a Sw Update update to function. You MUST put the private key and password file inside of the `repo-root-directory/keys` directory. The build will break if you do not complete this step and your GitHub bug will elicit an exasperated sign from the developers of Device Update. Be warned. 
```

Comment:
Re this: _and your GitHub bug will elicit an exasperated sign from the developers of Device Update. Be warned._ It's hilarious (and probably a true statement) but I think probably crosses the line from a tonal standpoint.

My main concern in cases like this is that the person reading this might be stressed due to a tight deadline, and may have had a bumpy start to the process by the time they get here. So humor may or may not land well.

Reply:
Fair. I'll edit it.